projects/plugins/backup/readme.txt

```diff
@@ -8,29 +8,152 @@ Stable tag: 0.1.0
 License: GPLv2 or later
 License URI: http://www.gnu.org/licenses/gpl-2.0.html
 
-Easily restore or download a backup of your site from a specific moment in time.
+Jetpack is the most proven WordPress backup plugin with 269 million site backups over the last ten years. Save every change and get back online quickly with one‑click restores.
```

Comment:
That's going to be too long for the short description. You can only use up to 151 characters here, the rest will be cut off.

Reply:
Thanks, @jeherve. I'll create another PR to fix that.